src/commands/suspend.js

```diff
@@ -0,0 +1,90 @@
+const Discord = require('discord.js');
+const getConfig = require('../config/get-config.js');
+const config = getConfig();
+module.exports = {
+  prefix: 'suspend',
+  description: 'Suspends a user for the given reason',
+  command: async function suspend(message) {
+    //check for appropriate permissions
+    if (message.member.hasPermission('KICK_MEMBERS') == false) {
+      message.channel.send(
+        `ERROR 401: ${message.author}, missing permissions.`
+      );
```

Comment:
Should we return errors back to the client like this into chat? Seems like spam. We should definitely log these errors via `console.error` or `console.log` if there is something wrong.

Reply:
I wrote this check as a client message because it indicates that the bot is still alive, but I can turn it to  a silent console.error if you'd rather.

Reply:
Changed to a `console.log`